resources/prometheus/prometheus-rules.yaml

```diff
@@ -291,3 +291,157 @@ spec:
+        # This is a time series of 0s (down) and 1s (up).
+        - expr: |
+            clamp_max(sum by(namespace) (kube_pod_container_status_ready{container="central", pod=~"central-.*", namespace=~"rhacs-.*"}), 1) or vector(0)
+          record: central:sli:pod_ready
+
+        # Error rate SLI
+        # The error rate over the last 10 minutes must be smaller than 35% to count as available.
+
+        # GRPC
+        - expr: |
+            sum by (namespace, rhacs_instance_id, rhacs_org_id, rhacs_org_name, rhacs_cluster_name, rhacs_environment)
+            (rate(grpc_server_handled_total{namespace=~"rhacs-.*", job="central", grpc_type="unary", grpc_code="OK"}[10m]))
+          record: central:grpc_server_handled:OK:rate10m
+
+        - expr: |
+            sum by (namespace, rhacs_instance_id, rhacs_org_id, rhacs_org_name, rhacs_cluster_name, rhacs_environment)
+            (rate(grpc_server_started_total{namespace=~"rhacs-.*", job="central", grpc_type="unary"}[10m]))
+          record: central:grpc_server_started:total:rate10m
+
+        # HTTP
+        - expr: |
+            sum by (namespace, rhacs_instance_id, rhacs_org_id, rhacs_org_name, rhacs_cluster_name, rhacs_environment)
+            (rate(http_incoming_requests_total{namespace=~"rhacs-.*", job="central", code!~"5.."}[10m]))
+          record: central:http_incoming_requests:not_5xx:rate10m
+
+        - expr: |
+            sum by (namespace, rhacs_instance_id, rhacs_org_id, rhacs_org_name, rhacs_cluster_name, rhacs_environment)
+            (rate(http_incoming_requests_total{namespace=~"rhacs-.*", job="central"}[10m]))
+          record: central:http_incoming_requests:total:rate10m
+
+        # This is a time series of 0s (down) and 1s (up).
+        # Success rate above 65% is floored to 1.
+        # Success rate below 65% is floored to 0.
+        - expr: |
+            floor(
+              (
+                central:grpc_server_handled:OK:rate10m
+              + central:http_incoming_requests:not_5xx:rate10m
+              ) / (
+                central:grpc_server_started:total:rate10m
+              + central:http_incoming_requests:total:rate10m
+              )
+              + 0.35
+            )
+          record: central:sli:error_rate
+
+        # Availability SLI
+        # If any of the SLIs is violated, the service counts as unavailable.
+        # This is a time series of 0s (down) and 1s (up).
+        - expr: |
+            central:sli:pod_ready * on (namespace) group_right() central:sli:error_rate
+          record: central:sli:availability
+
+        - expr: |
+            avg_over_time(central:sli:availability[1h])
+          record: central:sli:availability:avg_over_time1h
+
+        - expr: |
+            avg_over_time(central:sli:availability[28d])
+          record: central:sli:availability:avg_over_time28d
+
+    - name: rhacs-central.slo
+      rules:
+        # Availability SLO
+        - expr: "0.99"
```

Comment:
Ah one more issue: The SLO target for Central is `99.95%`, not `99%` as for fleet-manager. So we need to adapt this number here, and then fix the numbers in the tests as well.

Reply:
Another option would be to keep the `99%` for now for Central as well, with the intention to increase the target gradually to `99.95%`. Reason being that I don't think `99.95%` is really realistic with the current Central architecture. What is your opinion on this?

Reply:
> The SLO target for Central is `99.95%`
Let's set it so and be reminded.

Reply:
No, let's keep 99%, as 99.95% seems unrealistic today.